Tests/OpenAPIGeneratorReferenceTests/SnippetBasedReferenceTests.swift

```diff
@@ -5473,6 +5673,21 @@ extension SnippetBasedReferenceTests {
         let (registerHandlersDecl, _) = try translator.translateRegisterHandlers(operations)
         try XCTAssertSwiftEquivalent(registerHandlersDecl, expectedSwift, file: file, line: line)
     }
+
+    func assertServerTranslation(
+        _ serverYAML: String,
+        _ expectedSwift: String,
+        accessModifier: AccessModifier = .public,
+        featureFlags: FeatureFlags = [],
+        file: StaticString = #filePath,
+        line: UInt = #line
+    ) throws {
+        continueAfterFailure = false
+        let server = try YAMLDecoder().decode(OpenAPI.Server.self, from: serverYAML)
+        let translator = try makeTypesTranslator(accessModifier: accessModifier, featureFlags: featureFlags)
+        let translation = translator.translateServers([server])
+        try XCTAssertSwiftEquivalent(translation, expectedSwift, file: file, line: line)
+    }
```

Comment:
```suggestion
    func assertServersTranslation(
        _ serversYAML: String,
        _ expectedSwift: String,
        accessModifier: AccessModifier = .public,
        featureFlags: FeatureFlags = [],
        file: StaticString = #filePath,
        line: UInt = #line
    ) throws {
        continueAfterFailure = false
        let servers = try YAMLDecoder().decode([OpenAPI.Server].self, from: serversYAML)
        let translator = try makeTypesTranslator(accessModifier: accessModifier, featureFlags: featureFlags)
        let translation = translator.translateServers(servers)
        try XCTAssertSwiftEquivalent(translation, expectedSwift, file: file, line: line)
    }
```

If you make this change, you can then pass an array of servers for tests, e.g.

```
- url: ...
  description: ...
- url: ...
```

Reply:
Hmm. I tried that and the decoder threw a decoding error about a dictionary or something…maybe I had the indentation wrong on a line and didn't notice. will try it again, thanks :)

Reply:
Ok yep I must've just had some weird indentation. That works now, I've introduced two tests for multiple servers, one with and one without the feature flag enabled.